cookiecutter.json

```diff
@@ -9,6 +9,6 @@
   "models": "Comma-separated list of models",
   "config_class_name": "{{ cookiecutter.app_name|replace('_', ' ')|title|replace(' ', '') }}Config",
   "version": "0.1.0",
-  "owner": "edx/platform-team",
+  "owner": "edx/devops",
```

Comment:
I'm just...uh...copying the IDA cookiecutter...

Reply:
🤷‍♀️